mbed_lstools/lstools_base.py

```diff
@@ -47,7 +49,7 @@ def wrapper(self, *args):
                 raise e
             lock.release()
         else:
-            self.err("timed_mbedls_lock", "Failed to acquired mock file lock in %d seconds!" % timeout)
+            logger.err("Failed to acquired mock file lock in %d seconds!" , timeout)
```

Comment:
@theotherjimmy I don't _think_ the `logger.err` is an actual function in `logging`. I think it's actually `logger.error`.

I'm going off this doc page: https://docs.python.org/2/library/logging.html#logging.Logger.error

Reply:
You're correct.